front/ticket.form.php

```diff
@@ -67,12 +67,35 @@
         ]);
 
         $group_ticket = new Group_Ticket();
+
+        // retrieve the current status of the ticket
+        if ($_SESSION['plugins']['escalade']['config']['ticket_last_status'] == -1) {
+            $previous_ticket_status = (new Ticket())->getById($tickets_id)->fields['status'];
```

Comment:
`getById` is a static method, we do not need to use `new` here:

```suggestion
            $previous_ticket_status = Ticket::getById($tickets_id)->fields['status'];
```